src/base/datetime/mobile-calendar/body/style.ts

```diff
@@ -6,12 +6,20 @@ kuc-base-mobile-datetime-calendar-body * {
     "Lucida Sans Unicode", Arial, Verdana, sans-serif;
```

Comment:
There is no language = en and ja at here ?

Reply:
Yes.  It's the same with kintone native.